.changesets/rename-heartbeats-to-cron-check-ins.md

```diff
@@ -0,0 +1,20 @@
+---
+bump: patch
+type: change
+---
+
+Rename heartbeats to cron check-ins. Calls to `Appsignal.heartbeat` and `Appsignal.Heartbeat` should be replaced with calls to `Appsignal.CheckIn.cron` and `Appsignal.CheckIn.Cron`, for example:
+
+```elixir
+# Before
+Appsignal.heartbeat("do_something", fn ->
+  do_something()
+end)
+
+# After
+Appsignal.CheckIn.cron("do_something", fn ->
+  do_something
+end)
+```
+
+Calls to `Appsignal.heartbeat` and to methods in `Appsignal.Heartbeat` will emit a deprecation warning at compile-time.
```

Comment:
We can add this as a separate deprecation changeset so it shows up in the deprecations section of the changelog.